ghost/core/test/e2e-webhooks/pages.test.js

```diff
@@ -12,9 +12,21 @@ const {
     anyISODateTime,
     anyObjectId,
     anyContentVersion,
-    anyNumber
+    anyNumber,
+    anyString
 } = matchers;
 
+const tagSnapshot = {
+    created_at: anyISODateTime,
+    id: anyObjectId,
+    name: anyString,
```

Comment:
The tag's name, slug, visibility etc. are not changing between test runs? If they are not, would be best to have snapshot capture them as is - no need to include non-dynamic properties in the snapshot matcher object 👍